docs/Training_data_sources.md

```diff
@@ -0,0 +1,98 @@
+
+In the notebook `notebooks/Training data analysis.ipynb` I had a look at the extent and effect of the different sources of training data. I wanted to answer 3 broad questions:
+
+1. How much training data comes from each of the 4 tagging sources:
+- EPMC
+- Research Fish
+- Grants via Excel
+- Grants via Prodigy active learning
+2. How well does the 210223 ensemble do for each of these 4 sources?
+3. Does training the models with no EPMC or RF training data contributions improve the model?
+
+This analysis was performed on the following data:
+1. The training data that went into Prodigy : `../data/processed/training_data/210126/training_data.csv`
+2. The final Prodigy merged results (original training + new tags) : `../data/prodigy/merged_tech_grants/merged_tech_grants.jsonl`
+3. Which grants were from the test set for the 210221 model runs: `../data/processed/model_test_results/test_data_210221.csv`
+4. The predictions on all grants from the 210223 Ensemble model (which used the 210221 models): `../data/processed/ensemble/210223/wellcome-grants-awarded-2005-2019_tagged.csv`
+
+# 1. How much training data comes from each of the 4 tagging sources?
+
+The training data is labelled from:
+- Multiple sources: 26
+- RF: 57 (57 'tech')
+- EPMC: 126 (126 'tech')
```

Comment:
All of them that came from RF and EPMC are tech. Any specific reason why?

Reply:
These are the numbers that are linked back to grants, and we took the positives as evidence of a tech grant, but not the other way around i.e. if a paper didn't mention tech then we didn't take that as evidence that the grant didn't produce tech at all.